packages/ndla-notion/src/NotionBody.tsx

```diff
@@ -11,6 +11,15 @@ const NotionBody = styled.div`
     animation-name: fadeInLeft;
     animation-duration: 500ms;
   }
+  iframe {
+    height: 619px;
+    ${mq.range({ until: breakpoints.desktop })} {
+      height: 400px;
+    }
+    ${mq.range({ until: breakpoints.tabletWide })} {
+      height: 300px;
+    }
+  }
```

Comment:
Etter å ha tenkt litt mer på dette her tror jeg det greieste er å ikke endre på dette. Altså fjerne custom styling for iframe. H5P kan ha en hvilken som helst høyde, så tilpasning til H5Pen fra eksempelet vil uansett føre til feil.

Jeg testet et par inline begrepsforklaringer i test. Se (https://test.ndla.no/nn/article/31044). Her justeres høyden på iframen automatisk. Dette burde fungere tilsvarende i eksempelet.

Reply:
Se forklaring1 i lenken. Det er samme H5P.